src/metalperformanceshaders.cs

```diff
@@ -3178,9 +3174,6 @@ interface MPSCnnConvolutionState : MPSImageSizeEncodingState {
 
 		[Export ("kernelHeight")]
 		nuint KernelHeight { get; }
-
```

Comment:
I can't find this class anywhere, neither documentation, nor header. It's probably a good idea to remove the entire thing.

Reply:
It was inside `MPSNeuralNetwork.framework/Headers/MPSCNNConvolution.h` at some point but yeah I guess removing it will be better